gradle.properties

```diff
@@ -1,7 +1,9 @@
 #environment
 environmentName=GUI
+browserName=chrome
```

Comment:
You didn't modify the travis to change browserName to "headless"
I suggest you add the command via travis
or change this "browserName=`chrome`" value for `"headless"`

Reply:
I am working on travis separately in another branch. I will upload that my PR when everything is ok